day2/day2.py

```diff
@@ -16,11 +16,11 @@ def parse_args():
     return args
 
 
-def find_checksum():
+def find_checksum(checksum_list):
     chars = "abcdefghijklmnopqrstuvwxyz"
```

Comment:
`chars = string.ascii_lowercase`

Reply:
That's really cool, looks much cleaner !